client/src/components/List/List.jsx

```diff
@@ -1,10 +1,15 @@
-import React, { useState } from "react";
+import React, { useState,useContext } from "react";
 import Data from "../../data/businesses";
 import "./List.css";
+  // useContext
+import {BusinessContext} from "../BusinessContext/BusinessContext";
+
 export default function List() {
   //state for JSON data
   const [data, setData] = useState(Data);
```

Comment:
this can now be updated
```suggestion
  const { businesses, setBusinesses } = useContext(BusinessContext);
```

Reply:
Thank you! I fix the code.